### PR TITLE
Feature/ab2 d 5066 event service integration

### PR DESCRIPTION
**Pull request by sb-dkniesteadt**
**JIRA Tickets:**

[AB2D-5066](https://jira.cms.gov/browse/AB2D-5066) - Integrates ab2d with event service. Uses the event lib tools.
 
### What Does This PR Do?
Integrates ab2d with event service. Uses the event lib tools.

### What Should Reviewers Watch For?
Better ways to make calls. Test code
This is a large change. But most of of the changes are Identical.

### What Needs to Be Merged and Deployed Before this PR?
Event Variables. Will merge right before deployment.

### Submitter Checklist

- [ ] This PR includes any required documentation changes, (`README`, changelog / release notes, website).
- [ ] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments.
- [ ] Code checked for PHI/PII exposure

### CODE REVIEW

File: eventlogger/src/main/java/gov/cms/ab2d/eventlogger/events/ErrorEvent.java [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package gov.cms.ab2d.eventlogger.events;
 
-import gov.cms.ab2d.eventlogger.LoggableEvent;
```
Comment (sharshar-sb) [inline eventlogger/src/main/java/gov/cms/ab2d/eventlogger/events/ErrorEvent.java:3]: Why isn't this one deleted? Is it not in the event client?

File: worker/src/main/java/gov/cms/ab2d/worker/processor/ContractProcessorImpl.java [hunk 2]
```diff
@@ -49,7 +49,7 @@
 
 @Slf4j
 @Service
-@SuppressWarnings("java:S2142") //java:S2142: "InterruptedException" should not be ignored
+@SuppressWarnings({"java:S2142", "java:S3655"}) //java:S2142: "InterruptedException" should not be ignored
```
Comment (sharshar-sb) [inline worker/src/main/java/gov/cms/ab2d/worker/processor/ContractProcessorImpl.java:52]: Add what S3655 is in the comment like S2142 does.

File: worker/src/main/java/gov/cms/ab2d/worker/processor/JobPreProcessorImpl.java [hunk 3]
```diff
@@ -24,17 +24,18 @@
...
+import static gov.cms.ab2d.eventclient.events.SlackEvents.EOB_JOB_STARTED;
 import static gov.cms.ab2d.job.model.JobStatus.FAILED;
 import static gov.cms.ab2d.job.model.JobStatus.IN_PROGRESS;
 import static gov.cms.ab2d.job.model.JobStatus.SUBMITTED;
 import static gov.cms.ab2d.job.model.JobStatus.SUCCESSFUL;
 
 @Slf4j
 @Component
-@SuppressWarnings("java:S2142") //java:S2142: "InterruptedException" should not be ignored
+@SuppressWarnings({"java:S2142", "java:S2583"}) //java:S2142: "InterruptedException" should not be ignored
```
Comment (sharshar-sb) [inline worker/src/main/java/gov/cms/ab2d/worker/processor/JobPreProcessorImpl.java:38]: Same

**Review by sharshar-sb — CHANGES_REQUESTED** [review]

File: worker/src/main/java/gov/cms/ab2d/worker/processor/ContractProcessorImpl.java (hunk 2 above)
Comment (sb-dkniesteadt) [reply worker/src/main/java/gov/cms/ab2d/worker/processor/ContractProcessorImpl.java:52]: done

File: worker/src/main/java/gov/cms/ab2d/worker/processor/JobPreProcessorImpl.java (hunk 3 above)
Comment (sb-dkniesteadt) [reply worker/src/main/java/gov/cms/ab2d/worker/processor/JobPreProcessorImpl.java:38]: done

File: eventlogger/src/main/java/gov/cms/ab2d/eventlogger/events/ErrorEvent.java (hunk 1 above)
Comment (sb-dkniesteadt) [reply eventlogger/src/main/java/gov/cms/ab2d/eventlogger/events/ErrorEvent.java:3]: Made changes to event libs to resolve this. https://github.com/CMSgov/AB2D-Libs/pull/105

**Pull request closed without merging**